pkg/rngd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/containerd:5749f2e9e65395cc6635229e8da0e0d484320ddf AS mirror
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS mirror
 RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
 RUN apk add --no-cache --initdb -p /out \
```

Comment:
Wait - there's another containerd that needs to change on the second part of this build